commit 1578fcb4a0407cf368edb6ee4605c4ef9e77b6ed
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Tue Mar 20 00:06:46 2018 +0100

    Fix build with Qt 5.11 (missing headers)
    
    Reviewers: #kexi, staniek
    
    Reviewed By: #kexi, staniek
    
    Subscribers: staniek
    
    Tags: #kexi
    
    Differential Revision: https://phabricator.kde.org/D11503

diff --git a/src/main/startup/KexiNewProjectAssistant.cpp b/src/main/startup/KexiNewProjectAssistant.cpp
index 59e819acc..6568f1b5c 100644
--- a/src/main/startup/KexiNewProjectAssistant.cpp
+++ b/src/main/startup/KexiNewProjectAssistant.cpp
@@ -48,6 +48,7 @@
 #include <KIconLoader>
 #include <KStandardGuiItem>
 
+#include <QAction>
 #include <QDebug>
 #include <QLayout>
 #include <QCheckBox>
